scripts/fastsp_script/fsp_test.py

```diff
@@ -2,39 +2,98 @@
 import sys
 
 """
-###remember sys.argv - can create variables using indexes with sys.argv!  
-    
+###remember sys.argv - can create variables using indexes with sys.argv!      
 """
 
 dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
 slash = "/"
+fas = ".fasta"
 
-#print(sys.argv)
-
-outfile = output_fs.txt
-#will need to be .csv later 
+infile = open("output_fs.txt", "w+")
+infile_name = "output_fs.txt"
+outfile = open("parsed_fs_output.csv","w")
+header = "reference,estimation,datatype,sp_score,tc"+ "\n"
```

Comment:
FYI, can just include "\n" in the string to begin with: `header = "reference,estimation,datatype,sp_score,tc\n"`